src/main/java/com/google/javascript/clutz/ImportRenameMapBuilder.java

```diff
@@ -24,11 +24,14 @@
   public static Map<String, String> build(Collection<Node> parsedInputs) {
     Map<String, String> importRenameMap = new HashMap<>();
     for (Node ast : parsedInputs) {
-      // Symbols can be imported into a variable in a goog.module() file, so look for imports in the
-      // body of the goog module.
       String moduleId = getGoogModuleId(ast);
       if (moduleId != null) {
+        // Symbols can be imported into a variable in a goog.module() file, so look for imports in the
+        // body of the goog module.
         importRenameMap.putAll(build(moduleId, ast.getFirstChild()));
+      } else {
+        // Also get renames for symbols that are required, but not into a variable
```

Comment:
I don't get "required, but not into a variable"

Reply:
Rewritten - take another look?